### PR TITLE
more readable messages with increased line-height

### PR DESCRIPTION
**Pull request by tnguyen14**
Increasing the message `line-height` makes them a bit more readable for me.

I could foresee however that someone might prefer the current more compact setting of the messages. If that is so, I could create a setting that will switch the `line-height`, so the increased line-height could be by default or an opt-in setting.

### CODE REVIEW

**Comment by torhve** [comment]
I like our compact look in the default theme. 

I feel like changes like this is exactly why we support custom css, for those minute changes that personally feels nice.

But I would like for someone to create a less compact theme with more space all around.

**Comment by tnguyen14** [comment]
Yeah that makes sense.

I do have this style in my custom CSS. The reason I thought it would make sense to make it a configurable setting is when I look at some other chat tools like Slack or HipChat, most of them have the option to display more readable messages vs. a compact version.

**Comment by torhve** [comment]
We too have that option, but in a better form, fully replacable themes!

It's just that we haven't gotten many submissions for themes yet :-)

**Comment by tnguyen14** [comment]
If you think that this specific feature should be a theme, I could spin it off into one. Then perhaps others could chime in on where more spaces should be added. Right now, the only place that seems urgent to me is the messages.

If I was to do that, what theme should I base it off of? dark, black or light? Or should there be a less compact version for all 3?

**Comment by lorenzhs** [comment]
I think dark would be the best starting point, as it's our default theme and the most "polished" one (if you can say such a thing about our UI). The light theme needs work anyway, and black is only a dozen line change over dark, see https://github.com/glowing-bear/glowing-bear/blob/master/css/themes/black.css, so the changes to dark would carry over to black directly.

**Comment by tnguyen14** [comment]
@lorenzhs That makes sense to me. So should it be a new theme that extends `dark`, just like the black one, or should these changes be included directly into the `dark` theme?

**Comment by lorenzhs** [comment]
We'd like to keep the existing dark theme, I think a "spacious" theme would best be implemented in the same way as "black", by including the dark theme and building upon it. Does that sound workable?

**Comment by tnguyen14** [comment]
I've added a new `dark-spacious` theme to implement this change.

Any suggestions to other places where we can make it more "spacious" is welcome.

File: css/themes/dark-spacious.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import "dark.css";
```
Comment (lorenzhs) [inline css/themes/dark-spacious.css]: the correct syntax is `@import`, not `import`

**Comment by tnguyen14** [comment]
@lorenzhs thanks for pointing that out. my bad. I've fixed it.

**Comment by vrutkovs** [comment]
This makes nicks having the same line-height

Using ```tr.bufferline``` instead makes it much better

**Comment by lorenzhs** [comment]
hmm good idea, would you like to send a PR?

**Comment by tnguyen14** [comment]
@vrutkovs why would nicks having the same line-height bad?

**Comment by vrutkovs** [comment]
My fault, I meant to say that nick don't get 1.4 line-height

It gets displayed with a small vertical margin here:
selection_030

**Pull request merged**